plugins/content/course/model.schema

```diff
@@ -584,9 +584,22 @@
       "editorOnly": true,
       "inputType": "Checkbox",
       "validators": [],
-      "title": "Share with other users",
+      "title": "Share with all users",
       "help": "Controls whether or not your colleagues will be able to see this course from the 'Shared courses' option"
     },
+    "_shareWithUsers": {
```

Comment:
Adds extra sharing field

Reply:
TomG's PR is going to mean that these schemas get pulled in from the framework (https://github.com/adaptlearning/adapt_authoring/pull/2185) so I think that putting this here will mean that it is overwritten

Reply:
Have opened a framework PR here: https://github.com/adaptlearning/adapt_framework/pull/2346